api/cf-rpc-types/Cargo.toml

```diff
@@ -13,10 +13,12 @@ workspace = true
 [dependencies]
 anyhow = { workspace = true }
 serde = { workspace = true, default-features = true, features = ["derive"] }
+codec = { workspace = true, features = ["derive"] }
+scale-info = { workspace = true, features = ["derive"] }
```

Comment:
I think these just need `default-features = true` to be added.

Reply:
Unfortunately that didn't solve it... Same error:
```
error: the wasm*-unknown-unknown targets are not supported by default, you may need to enable the "js" feature. For more information see: https://docs.rs/getrandom/#webassembly-support
     --> /Users/mxu/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/getrandom-0.2.15/src/lib.rs:342:9
      |
  342 | /         compile_error!("the wasm*-unknown-unknown targets are not supported by \
  343 | |                         default, you may need to enable the \"js\" feature. \
  344 | |                         For more information see: \
  345 | |                         https://docs.rs/getrandom/#webassembly-support");
      | |________________________________________________________________________^

```